infra/benefit-finder-infra/.terraform-docs/footer.md

```diff
@@ -0,0 +1,76 @@
+### locals.tf Overview
+
+This is a high level overview of the `locals.tf` file. The locals.tf file itself is heavily commented and will go into detail about individual settings if further information is required.
+
+The locals.tf is the main file that needs to be edited to configure your infrastructure.
+
+####  Global variables
+
+##### project
+
+This variable holds the prefix of your resource names. For example, this project uses `vote` as a prefix for service names.
```

Comment:
are these references to `vote` true?

Reply:
Well, I am not referencing all the variables mentioned in the repo but still; let me clean up a little bit more. We wouldn't want reference to another project in our repo:)

Reply:
Resolved!